securitycenter/snippets_v2/snippets_bigquery_export_v2.py

```diff
@@ -0,0 +1,190 @@
+    bigquery_export.dataset = bigquery_dataset_id
+
+    request = securitycenter_v2.CreateBigQueryExportRequest()
+    request.parent = parent
+    request.big_query_export = bigquery_export
+    request.big_query_export_id = bigquery_export_id
+
+    # Create the export request.
+    response = client.create_big_query_export(request)
+
+    print(f"BigQuery export request created successfully: {response.name}\n")
+    return response
+
+# [END securitycenter_create_bigquery_export_v2]
+
+
+# [START securitycenter_get_bigquery_export_v2]
+def get_bigquery_export(parent: str, bigquery_export_id: str):
+    from google.cloud import securitycenter_v2
+
+    """
+    Retrieve an existing BigQuery export.
+    Args:
+        parent: Use any one of the following resource paths:
+                 - organizations/{organization_id}/locations/{location_id}
+                 - folders/{folder_id}/locations/{location_id}
+                 - projects/{project_id}/locations/{location_id}
+        bigquery_export_id: Unique identifier that is used to identify the export.
+    """
+
+    client = securitycenter_v2.SecurityCenterClient()
+
+    request = securitycenter_v2.GetBigQueryExportRequest()
+    request.name = f"{parent}/bigQueryExports/{bigquery_export_id}"
+
+    response = client.get_big_query_export(request)
+    print(f"Retrieved the BigQuery export: {response.name}")
+    return response
+
+# [END securitycenter_get_bigquery_export_v2]
+
+
+# [START securitycenter_list_bigquery_export_v2]
+def list_bigquery_exports(parent: str):
+    from google.cloud import securitycenter_v2
+
+    """
+    List BigQuery exports in the given parent.
+    Args:
+         parent: The parent which owns the collection of BigQuery exports.
+             Use any one of the following resource paths:
+                 - organizations/{organization_id}/locations/{location_id}
+                 - folders/{folder_id}/locations/{location_id}
+                 - projects/{project_id}/locations/{location_id}
+    """
+
+    client = securitycenter_v2.SecurityCenterClient()
+
+    request = securitycenter_v2.ListBigQueryExportsRequest()
+    request.parent = parent
+
+    response = client.list_big_query_exports(request)
+
+    print("Listing BigQuery exports:")
+    for bigquery_export in response:
+        print(bigquery_export.name)
+    return response
+
+# [END securitycenter_list_bigquery_export_v2]
+
+
+# [START securitycenter_update_bigquery_export_v2]
+def update_bigquery_export(parent: str, export_filter: str, bigquery_export_id: str):
+    """
+    Updates an existing BigQuery export.
+    Args:
+        parent: Use any one of the following resource paths:
+                 - organizations/{organization_id}/locations/{location_id}
+                 - folders/{folder_id}/locations/{location_id}
+                 - projects/{project_id}/locations/{location_id}
+        export_filter: Expression that defines the filter to apply across create/update events of findings.
```

Comment:
Add an example filter condition here, as in Java: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/security-command-center/snippets/src/main/java/vtwo/bigquery/UpdateBigQueryExport.java#L39-L42

Reply:
addressed